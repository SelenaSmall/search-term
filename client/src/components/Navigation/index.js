import React from 'react';
import { Link } from 'react-router-dom';
import {
  Collapse,
  Nav,
  Navbar,
  NavbarBrand,
  NavItem,
  NavLink,
} from 'reactstrap';

export default () => (
  <Navbar color="light" light expand="md">
    <NavbarBrand href="/" data-test-id="landing-page-link">
      Game
    </NavbarBrand>
    <Collapse navbar>
      <Nav className="ml-auto" navbar>
        <NavItem>
          <NavLink tag={Link} to="/profile" data-test-id="update-profile">
            Profile
          </NavLink>
        </NavItem>
      </Nav>
    </Collapse>
  </Navbar>
);
