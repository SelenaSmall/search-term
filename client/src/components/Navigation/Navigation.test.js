import React from 'react';
import { shallow } from 'enzyme';
import Navigation from '.';

it('renders without crashing', () => {
  const wrapper = shallow(<Navigation />);
  expect(wrapper).toMatchSnapshot();
});
