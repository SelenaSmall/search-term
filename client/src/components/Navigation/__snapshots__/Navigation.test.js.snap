// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Navbar
  color="light"
  expand="md"
  light={true}
  tag="nav"
>
  <NavbarBrand
    data-test-id="landing-page-link"
    href="/"
    tag="a"
  >
    Game
  </NavbarBrand>
  <Collapse
    appear={false}
    enter={true}
    exit={true}
    in={false}
    isOpen={false}
    mountOnEnter={false}
    navbar={true}
    onEnter={[Function]}
    onEntered={[Function]}
    onEntering={[Function]}
    onExit={[Function]}
    onExited={[Function]}
    onExiting={[Function]}
    tag="div"
    timeout={350}
    unmountOnExit={false}
  >
    <Nav
      className="ml-auto"
      navbar={true}
      tag="ul"
      vertical={false}
    >
      <NavItem
        tag="li"
      >
        <NavLink
          data-test-id="update-profile"
          tag={[Function]}
          to="/profile"
        >
          Profile
        </NavLink>
      </NavItem>
    </Nav>
  </Collapse>
</Navbar>
`;
